rpm/prebuild.sh

```diff
@@ -2,4 +2,17 @@
 
 set -exu  # Strict shell (w/o -o pipefail)
 
-curl -LsSf https://www.tarantool.io/release/1.10/installer.sh | sudo bash
+os=""
+dist=""
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
+    os="$ID"
+    dist="$VERSION_ID"
+fi
+
+if [[ "$os" == "almalinux" && "$dist" == 9* ]]; then
```

Comment:
Please, add a comment why do we need this `if`.

Reply:
Added.